samcli/local/docker/lambda_debug_settings.py

```diff
@@ -143,3 +149,12 @@ def get_debug_settings(debug_port, debug_args_list, _container_env_vars, runtime
                 LOG.debug("Passing entrypoint as specified in template")
                 return DebugSettings(entry + debug_args_list, _container_env_vars)
             raise DebuggingNotSupported("Debugging is not currently supported for {}".format(runtime)) from ex
+
+    @staticmethod
+    def parse_go_delve_api_version(debug_args_list: List[str]) -> int:
+        parser = ArgumentParser("Parser for delve args")
+        parser.add_argument("-delveApi", type=int, default=1)
```

Comment:
Makes sense, can you showcase the exact command that needs to be run to start a debugging session in the PR title, i.e what are the debug args you can pass in?

Reply:
it remains unchanged to the original sam-cli as illustrated in [this doc "Step-through debugging Golang functions locally"](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-using-debugging-golang.html):
only `-delveApi`

Reply:
Here is the example command from the doc: 
```bash
sam local start-api -d 5986 --debugger-path <delve folder path> --debug-args "-delveAPI=2"
```